assets/javascripts/discourse/components/chat-message.js

```diff
@@ -95,7 +95,8 @@ export default Component.extend({
     "canFlagMessage",
     "showDeleteButton",
     "showRestoreButton",
-    "showEditButton"
+    "showEditButton",
+    "showRebakeButton"
   )
   get moreButtons() {
```

Comment:
For consistency, I'd be great to convert that to a `discourseComputed`

Reply:
Yeah, discourseComputed should be applied to a regular method/function

Reply:
Done!